app/src/main/java/com/example/sarahtang/memberquiz/OtherActivity.java

```diff
@@ -0,0 +1,196 @@
+
+        countdowntimer = new CountDownTimer(6000, 1000) {
+
+            public void onTick(long millisUntilFinished) {
+                timer.setText("Time Left: " + millisUntilFinished / 1000);
+            }
+
+            public void onFinish() {
+                nextMember();
+                //timer.setText("done!");
+            }
+        };
+        countdowntimer.start();
+        nextMember();
+
+
+        endGame.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+                AlertDialog.Builder builder = new AlertDialog.Builder(OtherActivity.this);
+                builder.setMessage("Are you sure you want to exit?")
+                        .setPositiveButton("OK", new DialogInterface.OnClickListener() {
+                            public void onClick(DialogInterface dialog, int id) {
+                                Intent intent = new Intent(getApplicationContext(), MainActivity.class);
+                                startActivity(intent);
+                            }
+                        })
+                        .setNegativeButton("Cancel", null);
+
+                // Create the AlertDialog object and return it
+                builder.show();
+            }
+        });
+
+    }
+    private void nextMember() {
+        //calling for picture, each turn
+        String[] options = getOptions();
+        button2.setText(options[0]);
+        button3.setText(options[1]);
+        button4.setText(options[2]);
+        button5.setText(options[3]);
+        button2.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button3.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button4.setBackgroundColor(Color.parseColor("#fdfffc"));
+        button5.setBackgroundColor(Color.parseColor("#fdfffc"));
+        int currMemberIndex = (int) (Math.random() * options.length);
+        currMember = options[currMemberIndex];
+        String imgName = currMember.toLowerCase().replace(" ", "");
+        int id = getResources().getIdentifier(imgName,
+                "drawable", getPackageName());
+        memberPic.setImageResource(id);
+        countdowntimer.start();
+    }
+
+    private String[] getOptions() {
+        //Four button names things
+        String[] options = new String[4];
+        for (int i = 0; i < options.length; i++) {
+            while (options[i] == null) {
+                int random = (int) (Math.random() * members.length);
+                if (!containsMember(members[random], options)) {
+                    options[i] = members[random];
+                }
+            }
+        }
+        return options;
+    }
+
+    private boolean containsMember(String member, String[] options) {
+        //Does this contain member
+        for (int i = 0; i < options.length; i++) {
+            if (options[i] != null && options[i].equals(member)) {
+                return true;
+            }
+        }
+        return false;
+    }
+
```

Comment:
Didn't override onPause and onResume, timer will keep going even if I leave app!